README.md

```diff
@@ -16,6 +16,35 @@ For details about configuration and deployment of RBD and CephFS CSI plugins, se
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Support Matrix
```

Comment:
Reword this section, into multiple sections like so, (possibly drop the header "Ceph-CSI" in all)
- Ceph-CSI versions and CSI spec compatibility
- Ceph-CSI versions and Kubernetes compatibility
**NOTE:** there aren't too many versions yet, but CSI is not kubernetes specific, so we should delineate the CSI version table from the kubernetes version table
- Ceph-CSI features and available versions